packages/spatial/src/input/systems/ClientInputSystem.tsx

```diff
@@ -120,6 +120,7 @@ export function updateGamepadInput(eid: Entity) {
 const pointers = defineQuery([InputPointerComponent, InputSourceComponent, Not(XRSpaceComponent)])
 const xrSpaces = defineQuery([XRSpaceComponent, TransformComponent])
 const inputSources = defineQuery([InputSourceComponent, TransformComponent])
+const nonSpatialInputSources = defineQuery([InputSourceComponent])
```

Comment:
this isn't necessarily all non spatial input sources, rather ti is all input sources. this should be named `inputSourceQuery` and the one above should be called `spatialInputSourceQuery`

Reply:
addressed